docs/src/deployment.md

```diff
@@ -3,13 +3,15 @@ hide:
   - navigation
 ---
 
-The default runtimes are available through separate repositories ([eoapi-cdk](https://github.com/developmentseed/eoapi-cdk) and [eoapi-k8s](https://github.com/developmentseed/eoapi-k8s)). 
 
-A demonstration application is accessible with the repository [eoapi-template](https://github.com/developmentseed/eoapi-template).
+## Via [eoapi-cdk](https://github.com/developmentseed/eoapi-cdk)
 
-## AWS (Lambda)
 
-An example of Cloud Stack is available for AWS (RDS for the database and Lambda for the APIs)
+[eoapi-cdk](https://github.com/developmentseed/eoapi-cdk) is a set of AWS CDK constructs to deploy eoAPI services.
+
+[eoapi-template](https://github.com/developmentseed/eoapi-template) is an AWS CDK app that shows how to configure the [eoapi-cdk](https://github.com/developmentseed/eoapi-cdk) constructs.
+
+An example of Cloud Stack is available for AWS (RDS for the database and Lambda for the APIs).
```

Comment:
I think we should remove the following section and replace with a simple `eoapi-cdk` example

Reply:
I can't seem to find the equivalent for the eoAPI-k8s version. @alukach @sharkinsspatial could you put together a short getting started example for the eoapi-cdk repo?

Reply:
Sorry, I haven't really been following this project that closely.  Doesn't this repo ship with a `eoapi-cdk`-based example [here](https://github.com/developmentseed/eoAPI/blob/main/infrastructure/aws/cdk/app.py)?